javascript/ql/src/semmle/javascript/security/dataflow/ExceptionXss.qll

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about cross-site
+ * scripting vulnerabilities where the taint-flow passes through a thrown
+ * exception.
+ */
+
+import javascript
+
+module ExceptionXss {
+  import DomBasedXssCustomizations::DomBasedXss as DomBasedXssCustom
+  import ReflectedXssCustomizations::ReflectedXss as ReflectedXssCustom
+  import Xss::DomBasedXss as DomBasedXss
+  import Xss::ReflectedXss as ReflectedXSS
+  import Xss::StoredXss as StoredXss
+  import Xss as XSS
```

Comment:
Are we actually using all these imports? Also could you replace `XSS -> Xss` for consistency?

Reply:
I added all the imports to make sure that all sources/sinks of the other XSS queries where included. But some of them were redundant, so I removed those.